src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQueryOutputFormat.java

```diff
@@ -157,6 +160,9 @@ public void commitJob(JobContext jobContext) throws IOException {
       String tableKey = conf.get(BigQueryConstants.CONFIG_TABLE_KEY, null);
       tableKeyList = Arrays.stream(tableKey != null ? tableKey.split(",") : new String[0]).map(String::trim)
         .collect(Collectors.toList());
+      String orderedBy = conf.get(BigQueryConstants.CONFIG_ORDERED_BY, null);
+      orderedByList = Arrays.stream(orderedBy != null ? orderedBy.split(",") : new String[0]).map(String::trim)
+        .map(keyValue -> keyValue.replace(":", " ")).collect(Collectors.toList());
```

Comment:
the output format shouldn't need to replace any special characters. Whatever is configuring the outputformat should give well-formed values.

Reply:
Done.